### PR TITLE
Exclude unpublished courses

### PR DESCRIPTION
**Pull request by armstrongnate**
[ignore-commit-lint]

### CODE REVIEW

File: Core/Core/API/APICourseRequestable.swift [hunk 1]
```diff
@@ -24,13 +24,20 @@ public struct GetCoursesRequest: APIRequestable {
 
     public enum State: String {
         case available, completed, unpublished, deleted
+
+        public static var defaults: [State] {
+            if Bundle.main.isTeacherApp {
```
Comment (armstrongnate) [inline Core/Core/API/APICourseRequestable.swift]: Anyone have thoughts on how to override this for tests?

Comment (vanwagonet) [reply Core/Core/API/APICourseRequestable.swift]: Maybe `isTeacherApp` should also be true if the the main bundle is the runner?

Comment (armstrongnate) [reply Core/Core/API/APICourseRequestable.swift]: Good idea. Done.

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 94.98% | 94.86% | 0.12%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**